### DISCUSSION TITLE
trino can see mysql catalog & schema but not tables within it

### DISCUSSION THREAD

**Discussion by Slashoper** [trinodb/trino#17140] [Q&A]
![image](https://user-images.githubusercontent.com/19834014/233285397-79461fb2-cf05-4c1d-a6d0-3119bbc3f74a.png)
![image](https://user-images.githubusercontent.com/19834014/233285588-7cb5be19-6721-48fe-ad39-ccb44b46a6c3.png)
the trino-src and trino is both depoly on k8s . and trino-src user helm chart deploy ,using trino-402. the trino use trino-406
below is the  trino cluster web ui and the coordinator & work log . i found it normal .  i feel upset , i don't konw the reason
![image](https://user-images.githubusercontent.com/19834014/233286624-56e81859-424a-4031-9860-5f87199b224e.png)
![image](https://user-images.githubusercontent.com/19834014/233287096-8ec1098d-e87b-48f8-b8cf-fbaa9d161735.png)
![image](https://user-images.githubusercontent.com/19834014/233287713-b77786fc-e24e-49f4-ad61-ea5fc19ce9bc.png)
![image](https://user-images.githubusercontent.com/19834014/233288043-6cc6a615-50fd-465d-b98f-2baf01c6902a.png)

**Comment by Ftoy99** [comment]
Facing the same issue on aws EMR 6.10 it displays the catalog , schema but not the table inside the schema .
